Libplanet/Store/BlockSet.cs

```diff
@@ -4,23 +4,29 @@
 using System.Security.Cryptography;
 using Libplanet.Action;
 using Libplanet.Blocks;
+using LruCacheNet;
 
 namespace Libplanet.Store
 {
     public class BlockSet<T> : BaseIndex<HashDigest<SHA256>, Block<T>>
         where T : IAction, new()
     {
+        // FIXME it should be configurable
+        private const int DefaultCacheSize = 4096;
```

Comment:
How about turning this into a `readonly` field which is determined by a constructor argument so that it can be configured such as `new BlockSet<T>(cacheSize: 4096)`?

Reply:
I turned this value into `BlockSet<T>`'s constructor. https://github.com/planetarium/libplanet/pull/1013/commits/9545d572f9603d816ba9d65cccea1c870085c9a5